src/components/Reports/SalaryCalculator/Landing/NewSalaryCalculationLanding/NewSalaryCalculatorLanding.test.tsx

```diff
@@ -0,0 +1,189 @@
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {
+  AssignmentCategoryEnum,
+  PeopleGroupSupportTypeEnum,
+} from 'src/graphql/types.generated';
+import { LandingTestWrapper } from './LandingTestWrapper';
+import { NewSalaryCalculatorLanding } from './NewSalaryCalculatorLanding';
+
+interface TestComponentProps {
+  assignmentCategory?: AssignmentCategoryEnum;
+  peopleGroupSupportType?: PeopleGroupSupportTypeEnum;
+  hasInProgressCalculation?: boolean;
+  hasApprovedCalculation?: boolean;
+  onCall?: jest.Mock;
```

Comment:
We can define `mutationSpy` outside of the component and use it here instead of passing it in as a prop.